### PR TITLE
Add new user group management  in basic mode document

### PR DESCRIPTION
**Pull request by wutb15**
My document on user group management in basic authentication for the new design,please review.

### CODE REVIEW

File: docs/user_group_management.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  
+
+- User name ,password and email can be changed in webportal or using [rest apis](https://github.com/microsoft/pai/blob/master/docs/rest-server/API.md).
+
+  
+
+
+- Admin can also update users' grouplist in webportal .A user can belong to  more than one group in basic authentication mode.Admin can add a group or remove a group from users' grouplist.
+#### Delete a user
+- delete user using webportal or using [rest apis](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/microsoft/pai/master/src/rest-server/docs/swagger.yaml#tag/user)
```
Comment (abuccts) [inline docs/user_group_management.md]: change link to [deleteUser](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/microsoft/pai/master/src/rest-server/docs/swagger.yaml#operation/deleteUser) api

File: docs/user_group_management.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  
+
+### Group management
+
+  
+
+In basic authentication mode,a group named 'default'  and a group named 'admingroup' is created when deploy PAI. Every user needs to be in a group to have access to VCs and teamwise storages.
```
Comment (abuccts) [inline docs/user_group_management.md]: is created -> will be created

File: docs/user_group_management.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  It is possible to create a user without assigning any  group to it.This user will be assigned to group default. Groups that is admin  have access to all the VCs.
```
Comment (abuccts) [inline docs/user_group_management.md]: avoid multi-spaces

File: docs/user_group_management.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+
+  
+  
+  
+
+
+
+
+
```
Comment (abuccts) [inline docs/user_group_management.md]: same, remove

File: docs/user_group_management.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+### Group management
+
+
+In basic authentication mode, a group named 'default' and a group named 'admingroup' will be created when deploy OpenPAI. Every user needs to be in a group to have access to VCs and teamwise storages.
+It is possible to create a user without assigning any group to it. This user will be assigned to group default. Groups that are admin have access to all the VCs and teamwise storages.
+
+#### Create a new group
+
+In basic authentication mode, group is bind to virtual cluster. Admin can not create group directly in this mode. To create a new group, admin needs to do:
+- Create a new group by using [create group api](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/microsoft/pai/master/src/rest-server/docs/swagger.yaml#operation/createGroup) or webportal.
```
Comment (abuccts) [inline docs/user_group_management.md]: remove "or webportal"

File: docs/user_group_management.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+#### Update a group
+
+- Update teamwise storages, using [command line tool](https://github.com/microsoft/pai/blob/master/contrib/storage_plugin/README.MD) to update storage configs in the group.
+- Update VCs. Using [virtual cluster api](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/microsoft/pai/master/src/rest-server/docs/swagger.yaml#tag/virtual-cluster) to change the VCs that this group have access to.
+- Update adminrole. Using [group extension update api](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/microsoft/pai/master/src/rest-server/docs/swagger.yaml#operation/updateGroupExtensionAttribution) to update group's adminrole information,only users in admin group have access to administration.
+
+#### Delete a group
+
+- Use webportal or [delete group api](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/microsoft/pai/master/src/rest-server/docs/swagger.yaml#operation/deleteGroup) to remove a group from a cluster, the users in this group will be updated.
```
Comment (abuccts) [inline docs/user_group_management.md]: remove "webportal or"

File: docs/user_group_management.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+#### Delete a user
+- delete user using [delete user api](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/microsoft/pai/master/src/rest-server/docs/swagger.yaml#operation/deleteUser)
+
+
+### Group management
+
+
+In basic authentication mode, a group named 'default' and a group named 'admingroup' will be created when deploy OpenPAI. Every user needs to be in a group to have access to VCs and teamwise storages.
+It is possible to create a user without assigning any group to it. This user will be assigned to group default. Groups that are admin have access to all the VCs and teamwise storages.
```
Comment (Binyang2014) [inline docs/user_group_management.md]: group default -> group `default`.
Groups that are admin -> And `admingroup` has the permission to access all the VCs and ...

**Pull request merged**